scripts.d/s-salome-tar

```diff
@@ -164,6 +164,13 @@ function s-salome-tar-install()
 
     sed -i "s,https://www.salome-platform.org,https://salome.der.edf.fr,g" $LIGHTAPPGUI
     sed -i "s,https://discourse.salome-platform.org,https://forge.pleiade.edf.fr/projects/salome-rex/issues,g" $LIGHTAPPGUI
+
+    # Fix mesa_salome to link salome to native mesa
+
+    MESASALOME=$PREFIX/mesa_salome
+    LINENB=$(grep -Fn "#[mesa]" $MESASALOME | awk -F ":" '{print $1}')
+
+    sed -i "$((LINENB+1))d;$((LINENB+2))d;$((LINENB+4))d;$((LINENB+5))d" $MESASALOME
```

Comment:
This is very fragile as it depends on the number of lines in mesa_salome. We any update from the CEA adding more or removing some lines will possibly have disastrous consequences. Can you find a way to be immune to such changes? TIA.

Reply:
en fait, j'ai calcule le numero des lignes à partir de la ligne qui contient "#[mesa]". Donc normalement, ça va jamais planter sauf si CEA modifie la partie [mesa].

Reply:
Yes, and that's my fear here. We cannot have such a fragile code that can break at any point.
- What would happen if the section "#[mesa]" is not found?
- What would happen if the section contains more definitions
- What would happen if the section is changed to contain a single definition instead of 4 (if will remove some other stuf!).

What would be needed here:
- check for `#[mesa]`
- if found delete all lines until we get an empty one a a new section starting with `#[<some char>`